### PR TITLE
Edited Bootstrap gitops automation with Gimlet CLI Post

### PR DESCRIPTION
**Pull request by YoucefGuichi**

### CODE REVIEW

File: src/pages/docs/bootstrap-gitops-automation-with-gimlet-cli.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Bootstrap gitops automation with Gimlet CLI
+title: Bootstrap gitops automation with Gimlet CLI using one github repository
```
Comment (laszlocph) [inline src/pages/docs/bootstrap-gitops-automation-with-gimlet-cli.md:2]: Let's not change the title.

I see why you wanted to change it, cause we were having a consistency issue within the article.

I believe if we fix the article, the title will work okay.

File: src/pages/docs/bootstrap-gitops-automation-with-gimlet-cli.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Bootstrap gitops automation with Gimlet CLI
+title: Bootstrap gitops automation with Gimlet CLI using one github repository 
 description: In this guide you will use GimletCLI to bootstrap the gitops workflow, then write application manifests to the gitops repository and see it deploy.
```
Comment (laszlocph) [inline src/pages/docs/bootstrap-gitops-automation-with-gimlet-cli.md:3]: Can you change `GimletCLI` to `Gimlet CLI`?

File: src/pages/docs/bootstrap-gitops-automation-with-gimlet-cli.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
...
 ---
```
Comment (laszlocph) [inline src/pages/docs/bootstrap-gitops-automation-with-gimlet-cli.md:5]:
```suggestion
In this tutorial you will use the Gimlet CLI to bootstrap the gitops automation in a dummy repository. You will also write application manifests to this gitops repository and see them deploy.
```

File: src/pages/docs/bootstrap-gitops-automation-with-gimlet-cli.md [hunk 4]
```diff
@@ -116,6 +114,35 @@ First get some dummy yaml to deploy an Nginx container. Gimlet's OneChart Helm c
 ```
 helm template dummy-app -n staging onechart/onechart > manifest.yaml
 ```
+Or use the following manifest file:
```
Comment (laszlocph) [inline src/pages/docs/bootstrap-gitops-automation-with-gimlet-cli.md:117]: Can you remove the `helm template line ` completely? This tutorial should not rely on onechart knowledge.

File: src/pages/docs/bootstrap-gitops-automation-with-gimlet-cli.md [hunk 5]
```diff
@@ -116,6 +114,35 @@ First get some dummy yaml to deploy an Nginx container. Gimlet's OneChart Helm c
...
+    metadata:
+      labels:
+        application: dummy-app
+    spec:
+      containers:
+      - name: dummy-app
+        image: nginx
+        ports:
+          - containerPort: 80
+        resources:
```
Comment (laszlocph) [inline src/pages/docs/bootstrap-gitops-automation-with-gimlet-cli.md:141]: Remove the resources section, just to make the yaml shorter.

**Review by laszlocph — CHANGES_REQUESTED** [review]
Thank you for spotting the issue and making the article consistent.

I have a few comments.

File: src/pages/docs/bootstrap-gitops-automation-with-gimlet-cli.md (hunk 1 above)
Comment (laszlocph) [reply src/pages/docs/bootstrap-gitops-automation-with-gimlet-cli.md:2]: Can you change line 6 to this:

```
In this tutorial you will use the Gimlet CLI to bootstrap the gitops automation in a dummy repository. You will also write application manifests to this gitops repository and see them deploy.
```

Added `in a dummy repository` so it is perhaps more clear that it is a standalone tutorial, not following Gimlet gitops repo best practices.

File: src/pages/docs/bootstrap-gitops-automation-with-gimlet-cli.md [hunk 6]
```diff
@@ -76,20 +76,18 @@ Flux uses the `gitrepository` custom resource to point to git repository locatio
 
 ```bash
 ➜ kubectl get gitrepositories -A
-NAMESPACE     NAME                                         URL                                                   AGE   READY   STATUS
-flux-system   gitops-repo-gimlet-io-gitops-deleteme-apps    ssh://git@github.com/gimlet-io/gitops-deleteme-apps    60s   True    stored artifact for revision 'main/bb32202a9968cc290ff757f2a75bb17863d46e6e'
-flux-system   gitops-repo-gimlet-io-gitops-deleteme-infra   ssh://git@github.com/gimlet-io/gitops-deleteme-infra   60s   True    stored artifact for revision 'main/c29545f11e677479a44cfad85549b3b92af0a3c2'
+NAMESPACE     NAME                                                    URL                                                              AGE    READY   STATUS
+flux-system   gitops-repo-gimlet-boostraping-tutorial   ssh://git@github.com/gimlet/gimlet-boostraping-tutorial   125m   True    stored artifact for revision 'main/f4a2a676bbcc04f38120b24463ca1c66cc099ab4'
```
[the remark this answers is not in the document]
Comment (YoucefGuichi) [reply src/pages/docs/bootstrap-gitops-automation-with-gimlet-cli.md:80]: ok

**Review by laszlocph — APPROVED** [review]

**Pull request merged**